src/main/webapp/app/entities/flair-bi/flair-bi.html

```diff
@@ -178,7 +178,7 @@
                                         </svg>
                                     </span>
                                 </a>
-                                <a ng-show="vm.ngIfSettings()" href class="viz-header-button" ng-click="vm.flipCard(v)">
```

Comment:
keep the name as it is

Reply:
now I have used ngIfSettingsTable function here not ngIfSettings